Formula/eject.rb

```diff
@@ -0,0 +1,48 @@
+class Eject < Formula
+  desc "Generate swift code from Interface Builder xibs"
+  homepage "https://github.com/Raizlabs/Eject"
+  url "https://github.com/Raizlabs/Eject/archive/0.1.12.tar.gz"
+  sha256 "a4dae3d37f780d274f53ed25d9dc1a27d5245289f9b8cbaaf8be71bc9334de18"
+
+  depends_on :xcode => :build
```

Comment:
Failing on Yosemite so may need scoped to e.g. Xcode 8: https://bot.brew.sh/job/Homebrew%20Core/11454/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_eject/